src/app.module.ts

```diff
@@ -110,4 +51,67 @@ export class AppModule implements NestModule {
       .apply(ClsMiddleware, NotFoundLoggerMiddleware)
       .forRoutes({ path: '*', method: RequestMethod.ALL });
   }
+
+  static register(configFactory: ConfigFactory = configuration): DynamicModule {
+    return {
```

Comment:
@iamacook in https://github.com/safe-global/safe-client-gateway/pull/793 you will then be able to conditionally enable/disable the `AlertsModule` via this register function by accessing the `configuration()` object.